tests/v3/covid-19/api_gov.spec.js

```diff
@@ -71,6 +71,34 @@ describe.skip('TESTING /v3/covid-19/gov/canada', () => {
 	});
 });
 
+// here
```

Comment:
Put into own testing file

Reply:
You're right, this new variants endpoint needs its own test file, I'm working on that.